src/test/java/org/jabref/logic/importer/fetcher/MathSciNetTest.java

```diff
@@ -79,9 +76,25 @@ void searchByQueryFindsEntry() throws Exception {
     }
 
     @Test
-    @DisabledOnCIServer("CI server has no subscription to MathSciNet and thus gets 401 response")
-    void searchByIdFindsEntry() throws Exception {
-        Optional<BibEntry> fetchedEntry = fetcher.performSearchById("3537908");
-        assertEquals(Optional.of(ratiuEntry), fetchedEntry);
+    void getParser() throws Exception {
+        String fileName = "/importer/fetcher/jsonTest.json";
```

Comment:
Java has a special folder for resources src/main/resoures and for test `src/test/resources` See the other importer test files there